src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -26,57 +26,69 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
+import java.util.*;
+import java.lang.String;
 
 /** Provides access to the data stored in Datastore. */
 public class Datastore {
 
-  private DatastoreService datastore;
+		private DatastoreService datastore;
```

Comment:
Is there a reason for the extra white space? I think we should keep it to 2 spaces per indentation level, as per the [Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation)
```suggestion
  private DatastoreService datastore;
```

Reply:
It must have done that when I auto indented I'll fix it and re commit